index.js

```diff
@@ -1,39 +1,604 @@
+      inventory.includes("pizza")
+    ) {
+      console.log("You already have pizza.");
+      turnTaken = 0;
+    } else if (
+      simplifyString(answer).includes("leave") ||
+      simplifyString(answer).includes("exit") ||
+      simplifyString(answer) === "go outside" ||
+      simplifyString(answer) === "go main st" ||
+      simplifyString(answer) === "enter main st"
+    ) {
+      enterRoom("182 Main St");
+      console.log("You walk back out onto Main St.");
+    } else {
+      console.log("Sorry, I don't know how to \"" + answer + '".');
+      turnTaken = 0;
+    }
+  } else if (currentLocation === "Classroom") {
+    if (
+      simplifyString(answer).includes("sit") ||
+      simplifyString(answer) === "attend lecture" ||
+      simplifyString(answer) === "attend class"
+    ) {
+      if (currentState === "ON FIRE!!!") {
+        console.log(
+          "If you sit down on this nice chair while you're on fire, you'll ruin the chair. You can't bring yourself to be that person."
+        );
+        turnTaken = 0;
+      } else if (classTaken === 1) {
+        console.log(
+          "Class already happened. There's not much point in sitting down."
+        );
+        turnTaken = 0;
+      } else {
+        console.log(hungryMessage);
+        classTaken = 1;
+        enterState("Hungry");
+      }
+    } else if (
+      simplifyString(answer).includes("leave") ||
+      simplifyString(answer).includes("exit") ||
+      simplifyString(answer) === "go foyer" ||
+      simplifyString(answer) === "enter foyer"
+    ) {
+      enterRoom("Foyer");
+      console.log("You head back out into the foyer.");
+    } else if (simplifyString(answer) === "talk to josh") {
+      console.log("Josh says 'Hi.'");
+    } else {
+      console.log("Sorry, I don't know how to \"" + answer + '".');
+      turnTaken = 0;
+    }
+  } else if (
+    simplifyString(answer) === "go bathroom" ||
+    simplifyString(answer) === "enter bathroom"
+  ) {
+    console.log("The bathroom is outside the classroom, in the foyer.");
+    turnTaken = 0;
+  } else {
+    console.log("Sorry, I don't know how to \"" + answer + '".');
+    turnTaken = 0;
+  }
+  if (turnTaken === 1 && currentState === "Hungry") {
+    hungryTurnsLeft--;
+    if (hungryTurnsLeft > 3) {
+      console.log("Your stomach grumbles. Pizza is on your mind.");
+    } else if (hungryTurnsLeft > 0) {
+      console.log("Stomach...So empty...Must...Get pizza...");
+    } else if (hungryTurnsLeft === 0) {
+      console.log("You couldn't eat the pizza in time...");
+      enterState("DEAD!");
+    }
+  }
+  if (turnTaken === 1 && currentState === "ON FIRE!!!") {
+    hp = Math.round(hp - 20);
+    console.log(
+      "OOF, ouch, being on fire hurts!! You know have " +
+        hp +
+        " health points remaining!"
+    );
+  }
+  if (hp <= 0) {
+    enterState("DEAD!");
+  }
+  if (currentState === "DEAD!") {
+    console.log("YOU DIED");
```

Comment:
You could offer additional dialog stating that the game has ended because you died. For example: game over.